addons/hr/models/res_users.py

```diff
@@ -334,3 +334,12 @@ def action_open_employees(self):
             'res_id': employees.id,
             'view_mode': 'form',
         }
+
+    @api.model
+    def _systray_get_allowed_records(self, res_model, res_ids):
```

Comment:
Seems this is not necessary if we correctly use check_access_rights_rules in base method ?

Reply:
Up (not sure if updated or not): this is probably not necessary as the base method does the same ?

Reply:
Yes indeed, now that we don’t use the search anymore. Thanks. I have removed completely the method and computed the allowed record directly in the method systray_get_activities.